src/components/Person/Person.jsx

```diff
@@ -1 +1,26 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    {person.age && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+    )}
+    {person.sex === 'f' ? (
+      <p className="Person__partner">
+        {`${person.partnerName} is my husband`}
+      </p>
+    ) : (
+      <p className="Person__partner">
+        {`${person.partnerName} is my wife`}
+      </p>
+    )}
+    {!person.isMarried && (
+      <p className="Person__partner">
+        I am not married
+      </p>
```

Comment:
<img width="326" alt="image" src="https://github.com/mate-academy/react_person/assets/70220667/d02c19ac-3d3e-473e-8a3c-654191a52fbc">
looks broken


decide before jsx what text will be rendered and just render in jsx. something like that

```
const personPartnerType = sex === 'm' ? 'wife' : 'husband';
const personPartner = isMarried ? `${partnerName} is my ${personPartnerType}` : 'I am not married'
```

Reply:
not fixed
![image](https://github.com/mate-academy/react_person/assets/96265844/cc2094a7-53b8-4fa8-8856-dbec2abdeb89)